sdk/servicebus/service-bus/src/sender.ts

```diff
@@ -49,8 +49,9 @@ export interface ServiceBusSender {
    * method to send.
    * @param options - Options bag to pass an abort signal or tracing options.
    * @return Promise<void>
+   * @throws `MessagingError` with the code `MessageTooLargeError` if the provided messages do not fit in a single `ServiceBusMessageBatch`.
```

Comment:
Shouldnt this be `MessageSizeExceeded`?

Reply:
I actually wanted to ask about this one. Should this be a `ServiceBusError` instead of a `MessagingError`? Currently it is a `MessagingError` with the code hardcoded as `MessageTooLargeError`. I thought _maybe_ this was because we're generating the error on the client-side instead of receiving it from the service but wasn't sure if we were making that distinction here intentionally.

Reply:
With the introduction of `ServiceBusError`, the user should never see `MessagingError` again

Reply:
I'll merge this PR, but please follow up with this MessagingError vs ServicebusError situation in a follow up PR

Reply:
Logged https://github.com/Azure/azure-sdk-for-js/issues/12573